src/lib/status/index.ts

```diff
@@ -0,0 +1,33 @@
+import axios from "axios";
+
+import { GELATO_RELAY_URL } from "../../constants";
+import { getHttpErrorMessage } from "../../utils";
+
+import { Service, TransactionStatus } from "./types";
+
+/**
+ * @param {string} taskId - Task Id
+ * @param {Service} [service] - Optional Service parameter: Service.Relay or Service.Relay_V2
+ * @returns {Promise<TransactionStatus | undefined>} Transaction status of the task id
+ *
+ */
+export const getTaskStatus = async (
+  taskId: string,
+  service?: Service
+): Promise<TransactionStatus | undefined> => {
+  try {
+    let path: string;
+    if (service) {
+      path = `${GELATO_RELAY_URL}/tasks/status/${service}/${taskId}`;
+    } else {
+      path = `${GELATO_RELAY_URL}/tasks/status/${taskId}`;
```

Comment:
this path is defaulting to `service="RelayV2"`, so we could use only this one.

Reply:
https://github.com/gelatodigital/relay-sdk/pull/5/commits/c3673510de14f5f41128e9e17d1fd9c2947ceb65